### PR TITLE
Example to show publishing metrics to Cloud Watch

### PR DESCRIPTION
**Pull request by kiranmeduri**
*Issue #, if available:*
#120 

*Description of changes:*
Using telegraf to publish Envoy metrics to CloudWatch. Telegraf is an agent for collecting, processing, aggregating, and writing metrics, see https://github.com/influxdata/telegraf.

In this example telegraf is setup as statsd-sink for Envoy using statsd input-plugin. Telegraf then publishes the metrics to AWS CloudWatch using CW output-plugin.

See https://github.com/kiranmeduri/aws-app-mesh-examples/blob/howto-metrics-cloudwatch/walkthroughs/howto-metrics-cloudwatch/README.md for more information.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by lavignes** [comment]
Curious as to why telegraf was chosen over [cloudwatch-agent with statsd](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-custom-metrics-statsd.html). This is good either way, but was there a problem using the CW agent?

**Comment by kiranmeduri** [comment]
> Curious as to why telegraf was chosen over [cloudwatch-agent with statsd](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-custom-metrics-statsd.html). This is good either way, but was there a problem using the CW agent?

AFAIK, cw-agent uses telegraf internally. Problem with cw-agent is documented https://github.com/aws/aws-app-mesh-roadmap/issues/61. Please correct me if that is not a problem now. Thanks!

File: walkthroughs/howto-metrics-cloudwatch/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Overview
+This tutorial provides a walkthrough of using App Mesh and publishing metrics to AWS CloudWatch. Following is the system being setup with this walkthrough.
+
+![System Diagram](./images/app.png "System Diagram")
+
+## Bootstrap
+
+1. Clone this repository and navigate to the walkthrough/howto-basics folder, all commands will be ran from this location
```
Comment (bcelenza) [inline walkthroughs/howto-metrics-cloudwatch/README.md]: This step references `howto-basics` instead of `howto-metrics-cloudwatch`

**Comment by kiranmeduri** [comment]
I am going to discard this for now to prefer basing on latest cwagent. will open a new PR that works with CW insights. Thanks for reviewing.

**Pull request closed without merging**